android/src/main/java/com/cloudwebrtc/webrtc/utils/RTCBluetoothManager.java

```diff
@@ -233,7 +233,8 @@ public State getState() {
   public void start() {
     ThreadUtils.checkIsOnMainThread();
     Log.d(TAG, "start");
-    if (!hasPermission(apprtcContext, android.Manifest.permission.BLUETOOTH)) {
+    if (!hasPermission(apprtcContext, android.Manifest.permission.BLUETOOTH) 
+        && !hasPermission(apprtcContext, android.Manifest.permission.BLUETOOTH_CONNECT)) {
```

Comment:
I think this needs to check if on Android 12 first? This value may not exist on lower API versions and crash.

Reply:
Yes you're right. I'll update. Thanks.